linchpin/provision/roles/azure/molecule/docker/molecule.yml

```diff
@@ -0,0 +1,47 @@
+dependency:
+  name: galaxy
+  options:
+    role-file: molecule/shared/requirements.yml
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: azure-f31
+    image: lp_fedora31:latest
+    pull: false
+  - name: azure-f30
+    image: lp_fedora30
+    pull: false
+    ## Remove 'image' above and uncomment the following to run systemd in Docker
```

Comment:
Can you remove the commented lines here?  I have to go back and do it for AWS too

Reply:
yes